<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StarTrackerInfo</class>
 <widget class="QDockWidget" name="StarTrackerInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>101</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>101</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="features">
   <set>QDockWidget::NoDockWidgetFeatures</set>
   <set>QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>Star Tracker</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>398</width>
         <height>77</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>

        <item>
         <widget class="QGroupBox" name="GraphicGroup">
          <property name="title">
           <string/>
          </property>
         </widget>
        </item>

        <item>
          <widget class="QGraphicsView" name="graphicsView"/>
        </item>

        <item>
          <widget class="QGraphicsView" name="graphicsView2"/>
        </item>

        <item>
         <widget class="QGroupBox" name="InfoGroup">
         </widget>
        </item>

        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>

       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionHead1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Head1 View</string>
   </property>
  </action>
  <action name="actionHead2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Head2 View</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionHead1</sender>
   <signal>toggled(bool)</signal>
   <receiver>graphicsView</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
  <connection>
   <sender>actionHead2</sender>
   <signal>toggled(bool)</signal>
   <receiver>graphicsView2</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
 </connections>
</ui>
